File: branches/Medusa/Source/Fantasy.Base/XSerialization/IXCollectionSerializer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Collections;
using System.Xml.Linq;

namespace Fantasy.XSerialization
{
    public interface IXCollectionSerializer
    {
        void Save(IServiceProvider context, XElement element, IEnumerable collection);
        IEnumerable Load(IServiceProvider context, XElement element);
    }
}

File: branches/Stheno/Fantasy.Base/src/fantasy/xserialization/IXCollectionSerializer.java
package fantasy.xserialization;

import org.jdom2.*;

import fantasy.IServiceProvider;

public interface IXCollectionSerializer
{
	public void Save(IServiceProvider context, Element element, @SuppressWarnings("rawtypes") Iterable collection);
	@SuppressWarnings("rawtypes")
	Iterable Load(IServiceProvider context, Element element);
}